pkg/provider/aws/execution_unit.go

```diff
@@ -50,19 +50,25 @@ func (a *AWS) GenerateExecUnitResources(unit *core.ExecutionUnit, result *core.C
 		dag.AddDependency2(lambdaFunction, role)
 		dag.AddDependency2(lambdaFunction, image)
 		return nil
+	case Kubernetes:
+		return nil
 	default:
 		log.Errorf("Unsupported type, %s, for aws execution units", execUnitCfg.Type)
 
 	}
 	return nil
 }
 
-// convertExecUnitParams transforms the execution units environment variables to a map of key names and their corresponding core.IaCValue struct
+// convertExecUnitParams transforms the execution units environment variables to a map of key names and their corresponding core.IaCValue struct.
 //
-// If an environment variable does not pertain to a construct and is just a key, value string, the resource of the IaCValue will be left null
+// If an environment variable does not pertain to a construct and is just a key, value string, the resource of the IaCValue will be left null.
 func (a *AWS) convertExecUnitParams(result *core.ConstructGraph, dag *core.ResourceGraph) error {
 	execUnits := core.GetResourcesOfType[*core.ExecutionUnit](result)
 	for _, unit := range execUnits {
+
+		if a.Config.GetExecutionUnit(unit.ID).Type == Kubernetes {
```

Comment:
Why is k8s being singled out here? (And could you add a comment about it?)

Reply:
because we actually put the environment variables in the chart as values already so all we need to do there is inject them with real values. At this point we arent actually creating any resource for eks exec units until we get the helm charts up and running so this will likely change, we just dont have it figured out yet.

If you still want me to comment i can but i figured that this will end up being different soon